openMVCamera.py

```diff
@@ -0,0 +1,56 @@
+
+import sensor
+import time
+import math
+import time
+import utime
+import image
+import sensor
+import pyb
+from pyb import UART
+uart = UART(3, 115200, timeout_char=0)
+treshold_yellowgoal=(42, 99, -33, 40, 10, 72)
+treshold_bluegoal=(13, 80, 47, -36, -78, -14)

```

Comment:
Since python isnt statically typed or doenst have a concept of memory allocation as c++ we dont have something as the constants values in c++, but the standard is to name the variables in UPPERCASE like this->
```suggestion
THRESHOLD_YELLOW_GOAL=(42, 99, -33, 40, 10, 72)
THRESHOLD_BLUE_GOAL=(13, 80, 47, -36, -78, -14)
```